aip/general/0140.md

```diff
@@ -127,8 +127,8 @@ code in some languages.
 ### Display names
 
 Many resources have a human-readable name, often used for display in UI. This
-field **must** generally be called `display_name`, and **must not** have a
-uniqueness requirement.
+field **should** be called `display_name`, and **should not** have a uniqueness
+requirement.
```

Comment:
In the future, maybe we can clarify that while this field may restrict inputs to be unique, it **must not** be used as a mechanism for unique look-up / retrieval. Perhaps in AIP-148.